### PR TITLE
Correct documentation capitalization (close #8884)

### LINKED ISSUE #8884: docs: correct capitalization for hasura features
### Is your proposal related to a problem?

The capitalization in docs is quite random at the moment. Sometimes, Hasura features are sometimes in capital letters, and sometimes not. 

### Describe the solution you'd like

Capitalize all Hasura features, e.g. "Actions", "Event Triggers" etc.

### Instructions

Please read carefully through the [Wiki on writing documentation](https://hasura.io/docs/wiki/), especially the [style guide](https://hasura.io/docs/wiki/style/index/).

### PR DESCRIPTION
**Pull request by 3t8**
### Description
Fix capitalization of Hasura and Hasura features throughout the documentation

### Related Issues
#8884 

### Solution and Design
Improved readability by capitalizing Hasura features

### Affected components

- [x] Docs

### Breaking changes

- [x] No Breaking changes:
- [ ] There are breaking changes

### CODE REVIEW

File: docs/docs/schema/postgres/remote-relationships/index.mdx [hunk 1]
```diff
@@ -38,19 +38,19 @@ See the following guides on how to create different types of remote relationship
...
+  database and batches calls to Remote Schemas to avoid the
   [n+1 problem](https://hasura.io/learn/graphql/intro-graphql/graphql-server/) . More improvements to performance are
   upcoming.
 - **Schema integrity & consistency**: Hasura has metadata consistency checks at every level. Whenever you add a table,
-  remote schema, or action, Hasura makes sure that the graph that is exposed is consistent, and that all the
+  Remote Schema, or action, Hasura makes sure that the graph that is exposed is consistent, and that all the
   relationships make sense at a type level. This helps you in creating robust workflows to test changes in your CI and
   making safe deployments of your unified graph.
-- **Data federation**: With remote joins, the join, authorization, and consistency checks of added sources all happen at
+- **Data Federation**: With remote joins, the join, authorization, and consistency checks of added sources all happen at
```
Comment (robertjdominguez) [inline docs/docs/schema/postgres/remote-relationships/index.mdx:54]:
```suggestion
- **Data federation**: With remote joins, the join, authorization, and consistency checks of added sources all happen at
```

**Review by robertjdominguez — CHANGES_REQUESTED** [review]
Thanks for this PR @3t8 🚀 

This is a huge improvement! I've suggested a change for `Data Federation` to change back to `Data federation` as this is a convention that is not Hasura-specific.

Additionally, we need to revert the changes in the `hasura-cli` directory, as these are automatically generated and will be overwritten. We can take these up internally 👍

**Comment by 3t8** [comment]
@robertjdominguez Updated with lowecase data federation

**Comment by robertjdominguez** [comment]
@3t8, thanks for this work! Your implementation took cleverness and time, and we appreciate that ❤️ 

While we're not going to be able to merge your changes right now, we'd like to send you some swag to say thanks 🔥 

@Stefmore02 will be in touch soon 🚀

**Comment by Stefmore02** [comment]
Hey @3t8 !

We truly appreciate your contribution to Hasura OSS during Hacktoberfest 2022! We'd love to send you some Hasura swag as a token of appreciation. Could you please send me an email to [estefani@hasura.io](mailto:estefani@hasura.io) with a reference of this PR so I can send you the swag form? ✨

**Pull request closed without merging**